assets/js/modules/ads/components/settings/SettingsEdit.stories.js

```diff
@@ -196,12 +193,9 @@ GTGEnabled.decorators = [
 		};
 
 		return (
-			<WithTestRegistry
-				callback={ setupRegistry }
-				features={ [ 'googleTagGateway' ] }
```

Comment:
It doesn't seem like `features` was migrated. This should be kept, it just needs moving to `parameters`.